python/pylibcudf/pylibcudf/libcudf/utilities/default_stream.pxd

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.
```

Comment:
Note that I named this file `python/pylibcudf/pylibcudf/libcudf/utilities/default_stream.pxd` because I confused myself with the C++ group thing. Maybe this should just go in `python/pylibcudf/pylibcudf/libcudf/utilities.pxd`?

Reply:
Yes, probably.